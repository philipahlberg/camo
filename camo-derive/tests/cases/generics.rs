use camo_derive::Camo;

#[derive(Camo)]
struct Foo<T> {
    foo: T,
}

fn main() {}
