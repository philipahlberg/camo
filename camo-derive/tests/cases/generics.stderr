error: `camo` does not support generics
 --> $DIR/generics.rs:4:11
  |
4 | struct Foo<T> {
  |           ^^^
